docs/changes.rst

```diff
@@ -161,6 +161,12 @@ Enhancements
 * Added `libxml2` operating system package as a dependency. It provides xmllint,
   which is used for testing.
 
+* Added catching of some exceptions M2Cryptro can raise that were not catched
```

Comment:
changed catched to caught.  Don't you love english?

Reply:
DONE